build/docker/zap_common.py

```diff
@@ -319,3 +332,47 @@ def zap_get_alerts(zap, baseurl, blacklist, out_of_scope_dict):
         alerts = zap.core.alerts(start=st, count=pg)
     logging.debug('Total number of alerts: ' + str(alert_count))
     return alert_dict
+
+
+def get_latest_zap_client_version():
+    version_info = None
+
+    try:
+        version_info = urlopen('https://pypi.python.org/pypi/python-owasp-zap-v2.4/json')
+    except Exception, e:
```

Comment:
Updated and checked that:

python build/docker/zap_common.py
python3 build/docker/zap_common.py

don't throw syntax errors.